src/main/java/com/microsoft/lst_bench/task/TaskExecutor.java

```diff
@@ -100,6 +101,17 @@ public void executeTask(Connection connection, TaskExec task, Map<String, Object
                     + statement.getStatement()
                     + "; error message: "
                     + e.getMessage();
+
+            if (e instanceof SQLException) {
+              SQLException sqlException = (SQLException) e;
+              loggedError +=
+                  "SQL State: "
+                      + sqlException.getSQLState()
+                      + " Error Code: "
+                      + sqlException.getErrorCode()
+                      + loggedError;
```

Comment:
you're already adding loggedError with += in the beginning?

Reply:
Fixed